physics/cam_diagnostics.F90

```diff
@@ -197,11 +203,19 @@ subroutine diag_init_dry(pbuf2d)
 !   use physics_buffer,     only: pbuf_set_field
 !-ebudget update (tuo c)
     use tidal_diag,         only: tidal_diag_init
-    use cam_budget,            only: cam_budget_em_snapshot, cam_budget_em_register, thermo_budget_history ! ebudget update
+    use cam_budget,         only: cam_budget_em_snapshot, cam_budget_em_register, thermo_budget_history ! ebudget update
+#ifdef OSLO_AERO
+    use oslo_aero_params,   only: nbmodes 
+#endif
 
     type(physics_buffer_desc), pointer, intent(in) :: pbuf2d(:,:)
 
     integer :: k, m, istage
+
+    integer           :: i          ! OSLO_AERO
```

Comment:
I'm not thrilled with single letter variables (hard to search for). Any reason we couldn't use `indx` or something similar?

Reply:
Changed it to indx.